modules/widgets/top-posts.php

```diff
@@ -455,14 +458,6 @@ function get_by_views( $count, $args ) {
 		 */
 		$days = (int) apply_filters( 'jetpack_top_posts_days', 2, $args );
 
-		if ( $days < 1 ) {
```

Comment:
This was originally added to handle cases when `$days` is zero or evaluates to `false` after passing it through the filter and being cast to an `int`.

We can't keep it as is if we want to be able to handle `-1`, but it'd be nice to keep something that would default to `2` when `$days` is zero or `false`.